From 18dca4d265df1791b408cd2b0e1f89fce3338f26 Mon Sep 17 00:00:00 2001
From: placiano <placiano80@gmail.com>
Date: Sun, 20 Sep 2015 18:22:26 +0200
Subject: [PATCH] SM-N910F_511: patch init.target.rc

---
 SM-N910F_511/init.target.rc | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/SM-N910F_511/init.target.rc b/SM-N910F_511/init.target.rc
index b24b7d4..394ce37 100755
--- a/SM-N910F_511/init.target.rc
+++ b/SM-N910F_511/init.target.rc
@@ -121,15 +121,15 @@ service qseecomd /system/bin/qseecomd
 service mpdecision /system/bin/mpdecision --avg_comp
    class main
    user root
-   disabled
-
-service thermal-engine /system/bin/thermal-engine
-   class main
-   user root
-   socket thermal-send-client stream 0666 system system
-   socket thermal-recv-client stream 0660 system system
-   socket thermal-recv-passive-client stream 0666 system system
-   group root
+#   disabled
+
+#service thermal-engine /system/bin/thermal-engine
+#   class main
+#   user root
+#   socket thermal-send-client stream 0666 system system
+#   socket thermal-recv-client stream 0660 system system
+#   socket thermal-recv-passive-client stream 0666 system system
+#   group root
 
 service security-check1 /sbin/security_boot_check system
     class core
--
libgit2 0.22.2

